packages/wrangler/pages/functions/buildWorker.ts

```diff
@@ -21,9 +21,7 @@ export function buildWorker({
   onEnd = () => {},
 }: Options) {
   return build({
-    entryPoints: [
-      path.resolve(__dirname, "../pages/functions/template-worker.ts"),
-    ],
+    entryPoints: [path.resolve(__dirname, "./template-worker.ts")],
```

Comment:
I wonder if `__dirname` returns a different result once this file is bundled?

In other words, the `__dirname` at runtime is `packages/wrangler/wrangler-dist` rather than the source path of `packages/wrangler/pages/functions`.

I note that this only came up when we were trying to run Pages in a Wrangler unit test, which is not bundled...